cmd/juju/application/bundle.go

```diff
@@ -29,6 +30,7 @@ import (
 
 	"github.com/juju/juju/api"
 	"github.com/juju/juju/api/application"
+	app "github.com/juju/juju/apiserver/facades/client/application"
```

Comment:
I don't really like this alias but it's there to maintain consistency with `deploy.go` (lives in same package)